### PR TITLE
fixed jenkinsfile

### PR DESCRIPTION
**Pull request by rishvantsingh**
I have fixed and updated the jenkinsfile.

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,10 +1,31 @@
 pipeline {
-    agent any
+    agent none
     stages {
-        stage('Print hello') {
+        stage('Build') {
+            agent {
+                docker {
+                    image 'python:2-alpine'
```
Comment (santosh) [inline Jenkinsfile:7]: Update the python version. Check Dockerfile in the root of the project for reference.

File: Jenkinsfile [hunk 2]
```diff
@@ -1,10 +1,31 @@
...
+                }
+            }
+            steps {
+                sh 'python main.py'
+        
+            }
+        }
+        stage('Test') {
```
Comment (santosh) [inline Jenkinsfile:15]: Test step will go before build, so that we can skip build if the test fails.

**Comment by santosh** [comment]
Good work! 👍🏽 

I've also left a comment on the issue I suppose you are working on.

File: Jenkinsfile (hunk 2 above)
Comment (rishvantsingh) [reply Jenkinsfile:15]: Ook ill do it right now

**Comment by rishvantsingh** [comment]
Hey I have updated the Jenkinsfile with the mandatory changes.
Meanwhile you can merge it to your branch .And i will keep on working if in future i have to add more steps to it.

**Comment by santosh** [comment]
Thanks @rishvantsingh. Before we merge this changes to master. Have you tested the changes locally?

**Comment by rishvantsingh** [comment]
Yes i have tested it locally. It was smooth.

File: Jenkinsfile [hunk 3]
```diff
@@ -1,10 +1,37 @@
...
         }
+
+         stage('Build') {
+            agent {
+                docker {
+                    image 'python:3-alpine'
+                }
+            }
+            steps {
+                sh 'python main.py'
```
Comment (santosh) [inline Jenkinsfile:29]: Please build the docker image here. `python main.py` is suppose to be invoked from the container.

**Comment by rishvantsingh** [comment]
See I am already using the python image so i dont need to build the image.It will run in the container of the image being used in jenkinsfile.

**Comment by rishvantsingh** [comment]
So its totally fine to merge it.It will work.And in future ill welcome any changes.

**Comment by santosh** [comment]
Thanks for your contribution, but your contribution does not aligns with goals of the project.

**Pull request closed without merging**